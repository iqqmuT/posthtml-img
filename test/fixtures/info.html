<div>
  <img src="https://www.wikipedia.org/portal/wikipedia.org/assets/img/Wikipedia-logo-v2.png">
</div>
<div>
  <img src="https://upload.wikimedia.org/wikipedia/commons/9/90/CERN-aerial_1.jpg">
</div>
