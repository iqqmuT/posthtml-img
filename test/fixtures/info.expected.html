<div>
  <img src="https://www.wikipedia.org/portal/wikipedia.org/assets/img/Wikipedia-logo-v2.png" width="200" height="183">
</div>
<div>
  <img src="https://upload.wikimedia.org/wikipedia/commons/9/90/CERN-aerial_1.jpg" width="4320" height="2432">
</div>
